tensorflow/python/keras/callbacks.py

```diff
@@ -1272,11 +1272,13 @@ def _should_save_on_batch(self, batch):
       return True
     return False
 
-  def _save_model(self, epoch, logs):
+  def _save_model(self, epoch, batch, logs):
     """Saves the model.
 
     Arguments:
         epoch: the epoch this iteration is in.
+        batch: the batch this iteration is in. `None` if the `save_freq`
+          is set to `epoch`.
```

Comment:
Can this fit in the previous line?

Reply:
No, will give pylint error. But I have changed it to resolve error.